src/js/service/PiskelApiService.js

```diff
@@ -1,3 +1,5 @@
+/* global PiskelApi*/
+/* eslint no-undef: "error"*/
```

Comment:
I don't think you need the second line here - it's actually redundant with the [new eslint config, here](https://github.com/piskelapp/piskel/blob/bcee24609f776212050990040ea76722ff7dbe55/src/js/.eslintrc#L19).